### DISCUSSION TITLE
about login page

### DISCUSSION THREAD

**Discussion by dengxuezhao** [FreshRSS/FreshRSS#4731] [Q&A]
I deployed the freshrss service on the server and completed the installation, but now the login administrator account prompts that the passwords mismatch,and the login page does not contain the option to reset the password. What should I do？

**Comment by aledeg** [comment]
What you're experiencing is described in the documentation. Look at the FAQ page located [here](https://freshrss.github.io/FreshRSS/en/users/07_Frequently_Asked_Questions.html#how-can-you-change-a-forgotten-password).
